gulpfile.js

```diff
@@ -129,6 +131,16 @@ gulp.task('watch', _.without(BUILD_TASKS, 'webpack'), function(cb) {
   gulp.watch(LESS_FILES, ['less']);
 });
 
+gulp.task('server', function () {
```

Comment:
Hmm would it be better to just add this to the `watch` task? That way one could just run `gulp watch` in one Terminal window rather than having to run `gulp watch` in one, and then `gulp server` in another? Or is it possible to do `gulp watch server` and have both run simultaneously?

We should also add information about how to run/access this server to the quick start in the README. I can do that in a separate commit if you want, or you can do it, whatever your preference is.